dwave/cloud/utils.py

```diff
@@ -867,11 +873,52 @@ def get_contrib_packages():
 def get_platform_tags():
     """Return a list of platform tags generated from registered entry points."""
 
-    fs = [ep.load() for ep in iter_entry_points('dwave.common.platform.tags')]
+    fs = [ep.load() for ep in entry_points(group='dwave.common.platform.tags')]
     tags = list(filter(None, [f() for f in fs]))
     return tags
 
 
+class VersionNotFoundError(Exception):
+    """Package version requirement not satisfied."""
+
+
+def get_distribution(requirement: Union[str, Requirement],
+                     prereleases: bool = True) -> Distribution:
+    """Returns :class:`~importlib.metadata.Distribution` for a matching
+    `requirement` specification.
+
+    Note: this function re-implements :func:`pkg_resources.get_distribution`
+    functionality for py38+ (including py312, where setuptools/pkg_resources
+    is not available by default).
+
+    Args:
+        requirement:
+            Package dependency requirement according to PEP-508, given as string,
+            or :class:`~packaging.requirements.Requirement`.
+        prereleases:
+            Boolean flag to control if installed prereleases are allowed.
+
+    Raises:
+        :class:`~importlib.metadata.PackageNotFoundError`:
+            Package by name not found.
```

Comment:
```suggestion
            Named package not found.
```

Is this the meaning?

Reply:
No. I believe every package has a name. This error is raised when a search for package by name fails. The alternative to search-by-name is search by full requirement that includes name and version specifier.